frontend/src/scenes/setup/PreflightCheck.js

```diff
@@ -0,0 +1,236 @@
+            </Space>
+            <Col xs={24} style={{ marginTop: 60 }}>
+                <h2 className="subtitle text-center space-top">
+                    We're glad to have you here! We'll now guide you to setting up PostHog.
+                </h2>
+            </Col>
+            <Row style={{ display: 'flex', justifyContent: 'center' }}>
+                <div style={{ display: 'flex', alignItems: 'center', flexDirection: 'column' }}>
+                    <img src={hedgehogBlue} style={{ maxHeight: '100%' }} />
+                    <Button type="default" data-attr="support" data-source="preflight">
+                        <a href="https://posthog.com/support" target="_blank" rel="noreferrer">
+                            Support
+                        </a>
+                    </Button>
+                    <div className="breadcrumbs space-top">
+                        <span className="selected">Pre-flight check</span> &gt; Event capture &gt; Team setup
+                    </div>
+                </div>
+                <div
+                    style={{
+                        display: 'flex',
+                        justifyContent: 'center',
+                        marginLeft: 32,
+                        flexDirection: 'column',
+                        paddingTop: 32,
+                    }}
+                >
+                    <Card style={{ width: 600, width: '100%' }}>
+                        <Row style={{ marginBottom: 16 }}>
+                            <b style={{ fontSize: 16 }}>
+                                Preflight check{' '}
+                                {state.mode && (
+                                    <span>
+                                        (
+                                        <span
+                                            style={{ textDecoration: 'underline', cursor: 'pointer' }}
+                                            onClick={() => handleModeChange(null)}
+                                        >
+                                            {state.mode}
+                                        </span>
+                                        )
+                                    </span>
+                                )}
+                            </b>
+                            <Button
+                                type="default"
+                                style={{ position: 'absolute', right: 16 }}
+                                data-attr="preflight-refresh"
+                                icon={<SyncOutlined />}
+                                onClick={() => window.location.reload()}
+                            >
+                                Refresh
+                            </Button>
+                        </Row>
+                        <div>
+                            Tell us what you plan to do with this installation to make sure your infrastructure is ready
```

Comment:
I'd go for "What's your plan for this installation? We'll make infrastructure checks accordingly." (shorter sentences)